examples/src/main/java/tachyon/examples/JournalCrashTest.java

```diff
@@ -0,0 +1,377 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.examples;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.cli.BasicParser;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.Version;
+import tachyon.client.TachyonFS;
+import tachyon.conf.TachyonConf;
+import tachyon.util.CommonUtils;
+
+/**
+ * Class to perform Journal crash test. Launch clients to generate the Journal and check status.
+ */
+public class JournalCrashTest {
+
+  /**
+   * The operation types to test.
+   */
+  enum ClientOpType {
+    /**
+     * Keep creating empty file.
+     */
+    CREATE_FILE,
+    /**
+     * Keep creating and deleting file.
+     */
+    CREATE_DELETE_FILE,
+    /**
+     * Keep creating and renaming file.
+     */
+    CREATE_RENAME_FILE,
+    /**
+     * Keep creating empty raw table.
+     */
+    // TODO: add it back when supporting raw table
+    //CREATE_TABLE,
+    // TODO: add more op types to test
+  }
+
+  /**
+   * The client thread class. Each thread hold a Tachyon Client and keep requesting to Master.
+   */
+  static class ClientThread implements Runnable {
+    /** Which type of operation this thread should do. */
+    private final ClientOpType mOpType;
+    /** The Tachyon Client hold by this thread. */
+    // TODO: use TachyonFileSystem instead of the deprecated TachyonFS
+    private final TachyonFS mTfs;
```

Comment:
Yes. I'll fix the current issues, address the comments, then use the new client.
After that I'll remove the 'WIP' tag of this PR.